### PR TITLE
#160815819 Configure and connect to the database

### PR DESCRIPTION
**Pull request by Kiptoo-kanda**
#### What does this PR do?
Connects to the database 
#### Description of Task to be completed?
The database model was edited, and a new database was configured. Psycopg2 is required for the database connections
#### What are the relevant pivotal tracker stories?
#160815819

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+@manager.command
+def createsuperuser():
+    """Create a superuser, requires email and password."""
+
+    email = prompt('superuser email')
+    confirm_email = prompt('confirm superuser email')
+
+    if not EMAIL_REGEX.match(email):
+        sys.exit('\n kindly provide a valid email address')
```
Comment (PaulKariukiRimiru) [inline manage.py]: Instead of doing this. Please return a response to the user with that message

File: manage.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+    if not email == confirm_email:
+        sys.exit('\n kindly ensure that email and confirm email are identical')
```
Comment (PaulKariukiRimiru) [inline manage.py]: Same thing here

File: manage.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+    password = prompt_pass('superuser password')
+    confirm_password = prompt_pass('confirm superuser password')
+
+    if len(password) < 8:
+        sys.exit('\n kindly ensure that the password is at leaast 8 characters long')
```
Comment (PaulKariukiRimiru) [inline manage.py]: Same thing here, too

**Comment by PaulKariukiRimiru** [comment]
Fix the failing tests

File: manage.py (hunk 3 above)
Comment (Kiptoo-kanda) [reply manage.py]: Ok, will do for all the above.

**Pull request closed without merging**